daemon/src/oracle.rs

```diff
@@ -344,15 +340,30 @@ impl xtra::Actor for Actor {
     }
 }
 
+impl Attestation {
+    pub fn new(attestation: olivia::Attestation) -> Self {
+        Self(attestation)
+    }
+
+    pub fn into_inner(self) -> olivia::Attestation {
+        self.0
+    }
+
+    pub fn id(&self) -> BitMexPriceEventId {
+        self.0.id
+    }
+}
+
```

Comment:
Separate commit?

Reply:
It is motivated by creating `olivia::Attestation` right? Could move the entire creation of this dedicated model into a separate commit. We are starting to gold-plate here though :D